src/Orc.CsvTextEditor/Controls/ViewModels/FindReplaceDialogViewModel.cs

```diff
@@ -0,0 +1,92 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="FindReplaceDialogViewModel.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.CsvTextEditor
+{
+    using System.Media;
+    using Catel.IoC;
+    using Catel.MVVM;
+    using Controls;
+    using Services;
+
+    public class FindReplaceDialogViewModel : ViewModelBase
```

Comment:
may be it should be a manager with method "CsvTextEditorToolManager.ApplyTool<FindReplaceCsvEditorTool>(scope)"?